okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -229,11 +229,19 @@ private void logHandler(Buffer b) {
         promise.fail(m);
         return;
       }
+      // Docker returns lines of JSON objects.. We just consider the first one
+      // which is all we need.
+      String line = body.toString();
+      int idx = line.indexOf('\n');
+      if (idx != -1) {
+        line = line.substring(0, idx);
+      }
       try {
-        JsonObject b = body.toJsonObject();
+        JsonObject b = new JsonObject(line);
         promise.complete(b);
       } catch (DecodeException e) {
         logger.warn("{}", e.getMessage(), e);
+        logger.warn("while decoding {}", line);
```

Comment:
```suggestion
        logger.warn("{} while decoding {}", e.getMessage(), line);
```

Reply:
oh. forgot that one.. But the message was also shown previous line.